src/Blocks/BlockEnchantmentTable.h

```diff
@@ -34,6 +34,11 @@ class cBlockEnchantmentTableHandler :
 		UNUSED(a_Meta);
 		return 29;
 	}
+
+	virtual double GetBlockHeight(void) override
+	{
+		return 12.0 / 16.0;
```

Comment:
*most* compilers. Because we cannot be sure of this and because of general structure I would like to have a constant list of all block heights